### PR TITLE
Embed SVGs into html

### PR DESCRIPTION
**Pull request by SergeyRyabinin**
*Issue #, if available:*

*Description of changes:*

*Check all that applies:*
- [x] Did a review by yourself.
- [ ] Added proper tests to cover this PR. (If tests are not applicable, explain.)
- [ ] Checked if this PR is a breaking (APIs have been changed) change.
- [ ] Checked if this PR will _not_ introduce cross-platform inconsistent behavior.
- [ ] Checked if this PR would require a ReadMe/Wiki update.

Check which platforms you have built SDK on to verify the correctness of this PR.
- [x] Linux
- [ ] Windows
- [ ] Android
- [ ] MacOS
- [ ] IOS
- [ ] Other Platforms

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/doxygen/doxygen_wrapper.py [hunk 1]
```diff
@@ -18,6 +20,11 @@
 
 
 class DoxygenWrapper(object):
+    SVG_IMAGE_PATTERN = re.compile(
+        """ src="(?P<svg_filename>[a-z0-9_]+\\.svg)" """
```
Comment (DmitriyMusatkin) [inline docs/doxygen/doxygen_wrapper.py:24]: are all filenames guaranteed to be lower case? no other weird chars besides _?

File: docs/doxygen/doxygen_wrapper.py [hunk 2]
```diff
@@ -167,6 +174,35 @@ def _cleanup_dir_by_extension(child_dir: Path, extension: str):
...
+        svg_content = "data:image/svg+xml;base64,"
+        with open(svg_filename) as svg_file:
+            file_content = svg_file.read()
+            svg_content += base64.b64encode(file_content.encode()).decode()
+        return svg_content
+
+    def _embed_svg_images(self, html_dir):
+        p = Path(html_dir)
+        html_files = p.glob(f"**/*.html")
+        files_to_remove = set()
```
Comment (DmitriyMusatkin) [inline docs/doxygen/doxygen_wrapper.py:188]: is it common for doc to share same file across many html pages? i.e. are we increasing size by a lot by instead embedding image into every page?

**Review by DmitriyMusatkin — APPROVED** [review]

File: docs/doxygen/doxygen_wrapper.py (hunk 1 above)
Comment (SergeyRyabinin) [reply docs/doxygen/doxygen_wrapper.py:24]: Yes, at least at this point from doxygen.
The key is to stay consistent and don't break pages (i.e. if file is not embedded - it won't get deleted).

File: docs/doxygen/doxygen_wrapper.py (hunk 2 above)
Comment (SergeyRyabinin) [reply docs/doxygen/doxygen_wrapper.py:188]: So far we get a single SVG per single Class doc page.
I haven't seen an example of generated shared images (non-shared non-generated are in the SVG_TO_KEEP list).

**Pull request merged**